### ISSUE TITLE
Have Stalebot mark things as stale instead of wontfix

### ISSUE THREAD

**Issue by DarrenCook** [pouchdb/pouchdb#8044]
It is very confusing going through issues, because so many are tagged with "wontfix" but with no comment to explain why. In my mind "wontfix" means the developers regard it as a feature-not-a-bug, or what is being asked is impossible for some reason. "wontfix" should always come with a reason.

It eventually dawned on me that it is the cursed StaleBot doing this.

I.e. it is not "wontfix". It is "this may or may not be a bug, but no-one has got around to investigating it further yet, or we are waiting for more information on how to reproduce it".

That is a bit wordy, so how about it uses the label "stale" instead?

This is the file that needs to change:
https://github.com/pouchdb/pouchdb/blob/master/.github/stale.yml

Here is an example of another project that made this change for the same reason:
https://github.com/PyGithub/PyGithub/commit/9d9ce34dea76175d15b07cb7ad124b635095edd7

**Comment by daleharvey** [comment]
So I would be happy renaming it stale, definitely more accurate
